LO2Mixin.py

```diff
@@ -20,8 +20,6 @@
 # beginning _lo2_<FrameworkListener>
 def _decorate(fn, child, name):
     def wrap(*a, **kw):
-        LO2Mixin.log_message('calling LO2 wrapper: '+name)
```

Comment:
I removed this log statement because it was being called so often and cluttering the log output.